expression/builtin_miscellaneous_vec_test.go

```diff
@@ -28,8 +32,12 @@ var vecBuiltinMiscellaneousCases = map[string][]vecExprBenchCase{
 	ast.IsIPv6: {
 		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETString}},
 	},
-	ast.Sleep: {},
-	ast.UUID:  {},
+	ast.Sleep: {
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETReal}, geners: []dataGenerator{
+			newRangeRealGener(0, 0.1, 0),
```

Comment:
Could you implement `selectRealGener` like `selectStringGener` and just select `0` and `0.000001`.
So that the test time will not be affected.

Reply:
Done